graphql_api/helpers/mutation.py

```diff
@@ -57,5 +58,20 @@ def authenticated_resolver(instance, info, *args, **kwargs):
     return authenticated_resolver
 
 
+def require_part_of_org(resolver):
+    def authenticated_resolver(queried_owner, info, *args, **kwargs):
+        current_user = info.context["request"].user
+        current_owner = info.context["request"].current_owner
+        if not current_user.is_authenticated or not current_owner:
```

Comment:
will current_user always be non-null? Wonder if we should use .get() here instead

Reply:
Updated to use .get